data/human/hails.txt

```diff
@@ -870,7 +870,6 @@ phrase "friendly civilian"
 		"Where do astronauts hang out? At the space bar!"
 		"What is a Syndicate salesman's favorite drink? Whatever the Syndicate tell him it is!"
 		"Why does the Navy hate neighbours? Because the grass is greener on the other side!"
-		"What has two heads, six feet, two arms and one tail? A person on a horse!"
```

Comment:
Why was this removed? "Feet" here is plural of "foot", not a measurement unit.

Reply:
This is a pun, because a person is six feet (units of measurement) tall but there are six feet (body parts) in this configuration. The former is a non-SI unit though :)

Reply:
way to remove a good pun.

Reply:
The reference to non-SI units here is, honestly, actually optional. The joke *can* be read and understood completely literally with the simple understanding of six feet only referring to the six feet that a horse + human would have. Given that the imperial unit reference is simply an extra layer of entertainment, I believe that this should be left in.

Reply:
Worth noting that, at six feet tall for a rider on a horse, either the person or the horse is quite small. Possibly both. A short horse is 1.4m (4.5 feet) at the shoulder. Most people have more than 1.5 feet of height in their torse+head.

Reply:
Yeah, the “six feet” in this pun has nothing to do with measurements of length.

Reply:
Readded in e0d71968